snapcraft/cli/lifecycle.py

```diff
@@ -175,11 +177,12 @@ def clean(parts, step, **kwargs):
     """
     project_options = get_project_options(**kwargs)
     container_config = env.get_container_config()
+    step = step or 'pull'
```

Comment:
You can fix LP: [#1727445](https://bugs.launchpad.net/snapcraft/+bug/1727445) right here by not doing this.